frontend/src/lib/utils/authenticationStore.ts

```diff
@@ -1,17 +1,57 @@
 import {authenticate, me} from '$lib/api/candidate';
 import {writable, type Writable} from 'svelte/store';
 
-export type User = Record<string, string | number | boolean> | null;
+export interface User {
+  id: number;
+  username: string;
+  email: string;
+  confirmed: boolean;
+  blocked: boolean;
+  candidate: Candidate;
```

Comment:
This is an optional relation, so not sure if we want to consider defining it as `candidate: Candidate?` instead if the interface is also used somewhere else? (same thing applies for any other relation in this file)